docs/src/types/rode_types.md

```diff
@@ -41,7 +41,7 @@ page.
 * `p`: The optional parameters for the problem. Defaults to `NullParameters`.
 * `noise`: The noise process applied to the noise upon generation. Defaults to
   Gaussian white noise. For information on defining different noise processes,
-  see [the noise process documentation page](../../../features/noise_process)
+  see [the noise process documentation page](@ref noise_process)
```

Comment:
This is only correct if a label is put on the page, or it matches the title exactly.